source/content/guides/support/07-faq.md

```diff
@@ -20,13 +20,13 @@ reviewed: "2020-12-13"
 
 Yes. For more details, talk to your Customer Success Manager or [contact sales](https://pantheon.io/contact-us).
 
-### Is there a support number we can call? If so, is this service available for every package?
+### Is there a support number we can call? Is this service available for all account tiers?
 
-Diamond and Platinum customers have access to Pantheon On-Call, which includes an emergency 800-number to page an on-call engineer. For more details, [contact sales](https://pantheon.io/contact-us).
+Diamond and Platinum customers can call the 24/7 Premium Support Hotline for urgent account-related issues. You can find the phone number in the Support tab of your workspace.
 
-### If we open a ticket, do you provide 24/7 support for outages, or are there time restrictions?
+### Is there 24/7 support for emergencies like incidents or outages? Is this service available for all account tiers?
 
-Diamond, Platinum and Gold customers can file a downtime ticket for outages on our Support page, and we provide 24/7 on-call support for all platform issues affecting Diamond and Platinum customers.
+Diamond and Platinum customers can open an emergency ticket for business-critical issues. Pantheon escalates emergency tickets to our engineering team, even during off-hours. To ensure effective triage, please open the emergency ticket from the workspace of the affected site.
```

Comment:
Change `engineering team` to `support team`. We can also add links to the Workspace/Dashboard like we have in previous docs. Suggestion:
> Diamond and Platinum customers can open an emergency ticket for business-critical issues. Pantheon escalates emergency tickets to our support team, even during off-hours. To ensure effective triage, please create the emergency ticket from the Support page of either the [Workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces) or [Site Dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard) of the affected site.

Reply:
Same comment as above.